### PR TITLE
unit testing for add_team_context error handling

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -**  Added some unit testing for add_team_context error handling

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [ ] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [EDUCATOR-5290](https://openedx.atlassian.net/browse/EDUCATOR-5290)

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Review by Andytr1 — APPROVED** [review]
looks good

File: openassessment/xblock/test/test_submission.py [hunk 1]
```diff
@@ -566,6 +569,47 @@ def test_get_download_urls_from_submission_single_key(self, xblock):
...
+    @scenario('data/submission_open.xml', user_id="Red Five")
+    def test_get_team_context_exceptions(self, xblock):
+        """
+        Unit tests for error behavior in get_team_submission_context
+        """
+        xblock.location = 'some-uuid'
+        self.setup_mock_team(xblock)
+
+        # If there's no teams config, just return without adding anyting to the context, but log an error
+        with LogCapture() as logger:
```
Comment (muselesscreator) [inline openassessment/xblock/test/test_submission.py:581]: My only comment is that this should maybe be 2 tests for clearer test output, but is a nit.

**Review by muselesscreator — APPROVED** [review]
LGTM :shipit:

**Review by nsprenkle — APPROVED** [review]
LGTM, seconding @muselesscreator's comment.

**Pull request merged**